server/tests/conftest.py

```
@@ -1,22 +1,192 @@
+            "host": host,
+            "port": int(port),
+            "dbname": os.getenv("POSTGRES_DB", "aurora_db"),
+            "user": os.getenv("POSTGRES_USER", "aurora"),
+            "password": os.getenv("POSTGRES_PASSWORD", ""),
+            "sslmode": os.getenv("POSTGRES_SSLMODE", "prefer"),
+        }
+        sslrootcert = os.getenv("POSTGRES_SSLROOTCERT")
+        if sslrootcert:
+            params["sslrootcert"] = sslrootcert
+        return _psycopg2.connect(**params)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)